sdk/couchbase-core/src/queryx/query_options.rs

```diff
@@ -119,13 +118,15 @@ pub struct QueryOptions {
     pub read_only: Option<bool>,
     pub scan_cap: Option<u32>,
     pub scan_consistency: Option<ScanConsistency>,
-    pub scan_vector: Vec<u8>,
-    pub scan_vectors: HashMap<String, Vec<u8>>,
+    pub sparse_scan_vector: Option<SparseScanVectors>,
+    pub full_scan_vector: Option<FullScanVectors>,
+    pub sparse_scan_vectors: Option<HashMap<String, SparseScanVectors>>,
+    pub full_scan_vectors: Option<HashMap<String, FullScanVectors>>,
```

Comment:
OOI what are these scan vectors for?

Reply:
For scan consistency, read your own writes.